spec/models/placements/term_spec.rb

```diff
@@ -29,7 +29,7 @@
       let!(:summer_term) { create(:placements_term, :summer) }
 
       it "returns a collection of terms, in the order of Summer, Spring, Autumn" do
```

Comment:
Let's update the test description to match reality

```suggestion
      it "returns a collection of terms, in the order of Autumn, Spring, Summer" do
```